File: backend/src/main/java/com/ll/hotel/domain/hotel/option/dto/response/OptionResponse.java
package com.ll.hotel.domain.hotel.option.dto.response;

import com.ll.hotel.domain.hotel.option.entity.BaseOption;

public record OptionResponse(Long optionId, String name) {
    public static OptionResponse from(BaseOption option) {
        return new OptionResponse(option.getId(), option.getName());
    }
}

File: backend/src/main/java/com/ll/hotel/domain/hotel/option/dto/response/OptionResponse.kt
package com.ll.hotel.domain.hotel.option.dto.response

import com.ll.hotel.domain.hotel.option.entity.BaseOption

data class OptionResponse(
    val name: String,
    val optionId: Long
) {
    companion object {
        fun from(option: BaseOption) = OptionResponse(
                optionId = option.id,
                name = option.name
        )
    }
}